src/Compilers/CSharp/Portable/Emitter/Model/NamedTypeSymbolAdapter.cs

```diff
@@ -676,7 +679,7 @@ IEnumerable<Cci.IMethodDefinition> Cci.ITypeDefinition.GetMethods(EmitContext co
             {
                 foreach (var m in generated)
                 {
-                    if (m.ShouldInclude(context))
+                    if ((isAttribute && m.IsConstructor) || m.ShouldInclude(context))
```

Comment:
>IsConstructor [](start = 42, length = 13)

Can we use the same way to perform the constructor check here and above?

Reply:
Surprisingly, no - in the "normal" (not generated) method case, `method` is a `MethodSymbol`, but in the generated method case, `m` is a `Cci.IMethodDefinition`. It doesn't seem like there's a common API between them to check if it's a constructor.